src/printer.js

```diff
@@ -114,13 +114,13 @@ function genericPrint(path, options, printPath, args) {
   }
 
   if (needsParens) {
-    parts.unshift("( ");
+    parts.unshift("(", options.calypsoSpacing ? " " : "");
```

Comment:
About the naming: wouldn't `wpSpacing` and `wp-prettier` be better names than `calypsoSpacing` and `calypso-prettier`? This spacing style is really a WordPress thing, used also in non-Calypso code like the Gutenberg editor or JS in WordPress core.